src/jest.js

```diff
@@ -47,7 +47,15 @@ function getTestPaths(testRunner, config, patternInfo) {
   if (patternInfo.onlyChanged) {
     return findOnlyChangedTestPaths(testRunner, config);
   } else {
-    return testRunner.promiseTestPathsMatching(new RegExp(patternInfo.pattern));
+    return new Promise((resolve, reject) => {
+      fs.access(patternInfo.pattern, (err) => {
```

Comment:
oh no way! Next task: get rid of `existsSync` usage. I guess I'm fine with this then – but the slow test cache needs to be fixed :)